src/components/modal/modal.tsx

```diff
@@ -68,14 +68,8 @@ export class Modal implements ConditionalSlotComponent, OpenCloseComponent {
   //
   //--------------------------------------------------------------------------
 
-  /**
```

Comment:
Because I moved the prop to the interface, the doc got moved along with it. I just forgot that JSdoc needs to be in the TSX for the generated MD files. Sorry. Moving it back :)